src/cljs/athens/views/pages/core.cljs

```diff
@@ -51,6 +51,9 @@
 (defn view
   []
   (let [route-name (rf/subscribe [:current-route/name])]
+    ;; TODO: create a UI to inform the player of the connection status
+    (when (= @(rf/subscribe [:connection-status]) :reconnecting)
+      (rf/dispatch [:alert/js "Oops! Connection Lost. Reconnecting..."]))
```

Comment:
will this display alert every few seconds when we try to reconnect?

Reply:
It's good for now.

Reply:
> will this display alert every few seconds when we try to reconnect?

No, it's not.